tests/functional/test_e2e.py

```diff
@@ -19,29 +21,25 @@ def find_and_click(locator):
     """
     expect(locator).to_be_visible()
     locator.click()
```

Comment:
The wait_for_htmx is a good idea, but I think the existing code was actually redundant. [This](https://playwright.dev/docs/actionability) says that playwright auto waits for a locator to be visible, stable, enabled and receiving events before it attempts the click.